test/utils/utf8_demo.cpp

```diff
@@ -0,0 +1,222 @@
+// The content of the "UTF-8 encoded sample plain-text file" by Markus Kuhn.
```

Comment:
Do you have a link?

Reply:
I could not find the original file, but the content has license id and link to author's home page. Many projects uses this file, including:
- https://github.com/cyb70289/utf8
- https://www.w3.org/2001/06/utf-8-test/UTF-8-demo.html (older version).

Reply:
Actually, I have found it at https://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt.